headerCheck/factory/coreComponents.go

```diff
@@ -51,11 +56,28 @@ func CreateCoreComponents(
 		return nil, err
 	}
 
+	enableEpochsConfig.EnableEpochs.BLSMultiSignerEnableEpoch = []config.MultiSignerConfig{
```

Comment:
the epoch config should not be hardcoded here, as for different chains (devnet, mainnet, testnet) these could be different.

Reply:
for now i would hardcode them for different chains based on chainID, and put a TODO to expose them via proxy